src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/PcsVmrForwardFlower.cs

```diff
@@ -91,6 +91,7 @@ public async Task<bool> FlowForwardAsync(
             mapping,
             remotes,
             build.Commit,
+            ShouldResetBranchToRemoteWhenPreparingRepo(),
```

Comment:
Same, why not just pass `true`?

Reply:
I think I prefer having a property/method and just having a comment explaining it there